pkg/rbac/parser_integration_test.go

```diff
@@ -0,0 +1,57 @@
+package rbac_test
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/google/go-cmp/cmp"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"sigs.k8s.io/yaml"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+
+	"sigs.k8s.io/controller-tools/pkg/genall"
+	"sigs.k8s.io/controller-tools/pkg/loader"
+	"sigs.k8s.io/controller-tools/pkg/markers"
+	"sigs.k8s.io/controller-tools/pkg/rbac"
+)
+
+var _ = Describe("ClusterRole generated by the RBAC Generator", func() {
+	It("should match the expected result", func() {
```

Comment:
We may want to run the tests several times to ensure output is stable and no regression in the future.

Reply:
Added a retry loop for 5 times. Thanks!